### ISSUE TITLE
请将最新的代码发布至pypi

### ISSUE THREAD

**Issue by jhshengxy** [aliyun/django-oss-storage#20]
通过 pip install django-oss-storage 安装组建，目前pypi上的版本为1.1.0。
安装成功后，还是会出现
https://github.com/aliyun/django-oss-storage/issues/9
这个问题。

**Comment by favoyang** [comment]
The project is not very actively maintained. I believe [my bugfix branch](https://github.com/favoyang/django-oss-storage.git) covers most fixes at this moment (https://github.com/aliyun/django-oss-storage/issues/17, https://github.com/aliyun/django-oss-storage/issues/15, https://github.com/aliyun/django-oss-storage/issues/9). And based on jxltom's fixes. 

I'm currently using it in production.

```
pip install git+git://github.com/favoyang/django-oss-storage@bugfix#egg=django-oss-storage
```

**Comment by huiguangjun** [comment]
fixed in version 1.1.1
